lms/templates/learner_dashboard/program_details_sidebar.underscore

```diff
@@ -8,14 +8,61 @@
     <% } %>
 </aside>
 <aside class="aside js-course-certificates"></aside>
+<% if (isSubscriptionEligible) { %>
+<aside class="aside js-subscription-info subscription-info">
+    <h2 class="divider-heading">
+        <%- gettext(
+            hasActiveTrial
+            ? 'Trial subscription'
+            : subscriptionState === 'active'
+            ? 'Active subscription'
+            : 'Inactive subscription'
+        ) %>
+    </h2>
+    <div class="sidebar-section">
+        <div class="subscription-section">
+            <p class="my-0">
+                <%= HtmlUtils.interpolateHtml(
+                    gettext((
+                        subscriptionState === 'active'
+                        ? 'View your receipts or modify your subscription'
+                        : subscriptionState === 'inactive'
+                        ? 'Restart your subscription for {subscriptionPrice}/month. Your payment history is still available'
+                        : 'If you had a subscription previously, your payment history is still available'
+                    ).concat(' on the {a_start}Orders and subscriptions{a_end} page')),
```

Comment:
Will this result in there being multiple sentence fragments translated independently and then concatenated together? I worry about translation issues if that is the case.

Reply:
@justinhynes similar to your point on Learner Help Center needing to be capitalized, should the same go for "Orders and Subscriptions"?

Reply:
> Will this result in there being multiple sentence fragments translated independently and then concatenated together? I worry about translation issues if that is the case.

I believe it wouldn't do that since we concat before passing to gettext(the translation function) 
however we can change it to this if that's more readable/safe

```js
gettext((
     subscriptionState === 'active'
     ? 'View your receipts or modify your subscription on the {a_start}Orders and subscriptions{a_end} page'
     : subscriptionState === 'inactive'
     ? 'Restart your subscription for {subscriptionPrice}/month. Your payment history is still available on the {a_start}Orders and subscriptions{a_end} page'
     : 'If you had a subscription previously, your payment history is still available on the {a_start}Orders and subscriptions{a_end} page'
)),
```

Reply:
@jsnwesson about the capitalization part the design team confirmed all the text is capitalised as per edx standards

Reply:
Thanks for confirming the capitalization part! I will add that the change you suggested would be more readable at the very least, so it'd be appreciated.